### ISSUE TITLE
File extensions / JSX

### ISSUE THREAD

**Issue by jamiewinder** [rollup/rollup-plugin-typescript#11]
I'm porting a Typescript project over to Rollup, and am hoping to use this plugin to make the compile process straightforward. I'm facing two issues which I can't see referenced in the docs:
- JSX. I seem to get syntax errors on lines with JSX syntax. Is this supported?
- File extensions. It seems I have to alter my imports to have a file extension, otherwise I get an error saying the file wasn't found. Is there a way around this?

Here is my config:

``` javascript
var typescript = require('rollup-plugin-typescript');
var ts = require('typescript');

module.exports = {
    entry: 'src/index.tsx',
    dest: 'dist/test.js',
    format: 'es6',
    plugins: [
        typescript({
            target: ts.ScriptTarget.ES2015,
            jsx: "react"
        })
    ]
};
```

**Comment by Victorystick** [comment]
@jamiewinder Sorry for the delay. The issue of file extensions should be addressed by #17. I haven't gotten `ts.transpileModule` to accept `jsx` yet, but I'm on it.

**Comment by Victorystick** [comment]
The issue of file extensions should be fixed in version `0.3.0`. I'll create a new issue for JSX.
